src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -403,6 +403,17 @@ static bool operator<( const QVariant &a, const QVariant &b )
 }
 #endif
 
+#if QT_VERSION >= 0x050000 && QT_VERSION < 0x050600
```

Comment:
Hmm... maybe yes..

btw. looking at the code, just above the "less than" operator for QVariant should probably use qgsVariantLessThan?